x-pack/legacy/plugins/ml/public/data_frame/pages/transform_management/components/transform_list/transform_list.tsx

```diff
@@ -143,6 +145,25 @@ export const DataFrameTransformList: SFC = () => {
     hidePerPageOptions: false,
   };
 
+  const search = {
+    box: {
+      incremental: true,
+    },
+    filters: [
```

Comment:
I think it would be useful to add filter buttons for continuous or batch mode, as well as the Status one.

Reply:
Added in e44fd22a72262b9c9937130b8d77288bce98e0d7